### ISSUE TITLE
NormalizeImage 为什么加 1/255的scale

### ISSUE THREAD

**Issue by renmada** [PaddlePaddle/PaddleClas#1541]
好多模型的，比如mobilenetv2的config有加了scale，我看很多其他框架的训练代码没有scale。

在训练imagenet的时候发现验证集精度很低，会是因为这个造成的吗

confg如下
```
# global configs
Global:
  checkpoints: null
  pretrained_model: null
  output_dir: ./output/
  device: gpu
  save_interval: 1
  eval_during_train: True
  eval_interval: 1
  epochs: 240
  print_batch_step: 10
  use_visualdl: False
  # used for static mode and model export
  image_shape: [3, 224, 224]
  save_inference_dir: ./inference

# model architecture
Arch:
  name: OctMobileNetV2_x1_125
  class_num: 1000
 
# loss function config for traing/eval process
Loss:
  Train:
    - CELoss:
        weight: 1.0
        epsilon: 0.1
  Eval:
    - CELoss:
        weight: 1.0


Optimizer:
  name: Momentum
  momentum: 0.9
  lr:
    name: Cosine
    learning_rate: 0.04
  regularizer:
    name: 'L2'
    coeff: 0.00004


# data loader for train and eval
DataLoader:
  Train:
    dataset:
      name: ImageNetDataset
      image_root: ../ILSVRC2012_w/
      cls_label_path: ../ILSVRC2012_w/train_list.txt
      transform_ops:
        - DecodeImage:
            to_rgb: True
            channel_first: False
        - RandCropImage:
            size: 224
        - RandFlipImage:
            flip_code: 1
        - ColorJitter:
            brightness: 0.4
            contrast: 0.4
            saturation: 0.4
        - NormalizeImage:
            scale: 1.0/255.0
            mean: [0.485, 0.456, 0.406]
            std: [0.229, 0.224, 0.225]
            order: ''
      batch_transform_ops:
        - MixupOperator:
            alpha: 0.2

    sampler:
      name: DistributedBatchSampler
      batch_size: 384
      drop_last: False
      shuffle: True
    loader:
      num_workers: 4
      use_shared_memory: True

  Eval:
    dataset: 
      name: ImageNetDataset
      image_root: ../ILSVRC2012_w/
      cls_label_path: ../ILSVRC2012_w/val_list.txt
      transform_ops:
        - DecodeImage:
            to_rgb: True
            channel_first: False
        - ResizeImage:
            resize_short: 256
        - CropImage:
            size: 224
        - NormalizeImage:
            scale: 1.0/255.0
            mean: [0.485, 0.456, 0.406]
            std: [0.229, 0.224, 0.225]
            order: ''
    sampler:
      name: DistributedBatchSampler
      batch_size: 384
      drop_last: False
      shuffle: False
    loader:
      num_workers: 4
      use_shared_memory: True

Infer:
  infer_imgs: docs/images/whl/demo.jpg
  batch_size: 384
  transforms:
    - DecodeImage:
        to_rgb: True
        channel_first: False
    - ResizeImage:
        resize_short: 256
    - CropImage:
        size: 224
    - NormalizeImage:
        scale: 1.0/255.0
        mean: [0.485, 0.456, 0.406]
        std: [0.229, 0.224, 0.225]
        order: ''
    - ToCHWImage:
  PostProcess:
    name: Topk
    topk: 5
    class_id_map_file: ppcls/utils/imagenet1k_label_list.txt

Metric:
  Train:
    - TopkAcc:
        topk: [1, 5]
  Eval:
    - TopkAcc:
        topk: [1, 5]

```

训练日志
```
reader_cost: 4.52641, ips: 76.57218 images/sec
[2021/12/09 22:54:14] root INFO: [Eval][Epoch 14][Iter: 10/33]CELoss: 3.07738, loss: 3.07738, top1: 0.00000, top5: 0.00260, batch_cost: 0.77385s, reader_cost: 0.57224, ips: 496.22269 images/sec
[2021/12/09 22:54:26] root INFO: [Eval][Epoch 14][Iter: 20/33]CELoss: 2.93603, loss: 2.93603, top1: 0.00195, top5: 0.00456, batch_cost: 1.02122s, reader_cost: 0.78887, ips: 376.01989 images/sec
[2021/12/09 22:54:35] root INFO: [Eval][Epoch 14][Iter: 30/33]CELoss: 3.00043, loss: 3.00043, top1: 0.00260, top5: 0.00716, batch_cost: 0.95334s, reader_cost: 0.69650, ips: 402.79635 images/sec
[2021/12/09 22:54:36] root INFO: [Eval][Epoch 14][Avg]CELoss: 2.92478, loss: 2.92478, top1: 0.00112, top5: 0.00408
[2021/12/09 22:54:36] root INFO: [Eval][Epoch 14][best metric: 0.0031000000271201133]
[2021/12/09 22:54:37] root INFO: Already save model in ./output/OctMobileNetV2_x1_125/epoch_14
[2021/12/09 22:54:37] root INFO: Already save model in ./output/OctMobileNetV2_x1_125/latest
[2021/12/09 22:54:43] root INFO: [Train][Epoch 15/240][Iter: 0/835]lr: 0.03986, top1: 0.35417, top5: 0.58073, CELoss: 3.84646, loss: 3.84646, batch_cost: 1.07799s, reader_cost: 0.45171, ips: 356.21716 images/sec, eta: 2 days, 8:30:28
[2021/12/09 22:54:53] root INFO: [Train][Epoch 15/240][Iter: 10/835]lr: 0.03985, top1: 0.35440, top5: 0.58854, CELoss: 3.95077, loss: 3.95077, batch_cost: 0.90153s, reader_cost: 0.36024, ips: 425.94298 images/sec, eta: 1 day, 23:15:18
[2021/12/09 22:55:05] root INFO: [Train][Epoch 15/240][Iter: 20/835]lr: 0.03985, top1: 0.33718, top5: 0.57155, CELoss: 4.15590, loss: 4.15590, batch_cost: 1.11669s, reader_cost: 0.57560, ips: 343.87425 images/sec, eta: 2 days, 10:31:47
[2021/12/09 22:55:15] root INFO: [Train][Epoch 15/240][Iter: 30/835]lr: 0.03985, top1: 0.32149, top5: 0.55242, CELoss: 4.26796, loss: 4.26796, batch_cost: 1.05383s, reader_cost: 0.50752, ips: 364.38586 images/sec, eta: 2 days, 7:13:56
[2021/12/09 22:55:26] root INFO: [Train][Epoch 15/240][Iter: 40/835]lr: 0.03985, top1: 0.32495, top5: 0.55380, CELoss: 4.28579, loss: 4.28579, batch_cost: 1.07878s, reader_cost: 0.52657, ips: 355.95731 images/sec, eta: 2 days, 8:32:13
[2021/12/09 22:55:36] root INFO: [Train][Epoch 15/240][Iter: 50/835]lr: 0.03985, top1: 0.32465, top5: 0.55362, CELoss: 4.30194, loss: 4.30194, batch_cost: 1.05862s, reader_cost: 0.47323, ips: 362.73589 images/sec, eta: 2 days, 7:28:39
[2021/12/09 22:55:47] root INFO: [Train][Epoch 15/240][Iter: 60/835]lr: 0.03985, top1: 0.32731, top5: 0.55678, CELoss: 4.29902, loss: 4.29902, batch_cost: 1.05938s, reader_cost: 0.45758, ips: 362.47540 images/sec, eta: 2 days, 7:30:52
```

**Comment by RainFrost1** [comment]
`scale`是将原本图像0~255的区间映射到0～1的区间，现在所有框架的绝大部分模型应该都有`Normalize`预处理操作，可以具体看下

**Comment by renmada** [comment]
> `scale`是将原本图像0~255的区间映射到0～1的区间，现在所有框架的绝大部分模型应该都有`Normalize`预处理操作，可以具体看下

- 其他框架的Normalize都是不加1/255的scale的
- 另外还要个请求，能不能提供一份能收敛的imagenet的训练日志，模型不限。
用于训练imagenet的时候做对比，谢谢

**Comment by cuicheng01** [comment]
1.这个目的就是把0-255的数值映射到0-1，其他框架也有类似的操作，比如pytorch中的transforms.ToTensor() 操作；
2.模型的训练日志暂时没有很多，如果有需要，可以留一下邮箱地址，我们可以发送近期训练PPLCNet的log

**Comment by renmada** [comment]
> 1.这个目的就是把0-255的数值映射到0-1，其他框架也有类似的操作，比如pytorch中的transforms.ToTensor() 操作； 2.模型的训练日志暂时没有很多，如果有需要，可以留一下邮箱地址，我们可以发送近期训练PPLCNet的log

原来是在ToTensor的时候做的scale，没看那里的实现
邮箱：imxly2@126.com
谢谢!
